### DISCUSSION TITLE
How to Show Vercel Websites for Search Engines?

### DISCUSSION THREAD

**Discussion by kxzo** [vercel/community#815] [Help] [answered]
**Hey**, I gotta say I love vercel, I mean who doesn't!? Anyways, so I want to showcase my website that has a specific vercel domain such as: _somecoolappname.vercel.app_. How do I do that? I mean I have done some meta tags work for _document.js and now after deploying the newest build of my website, I enter the excact same keywords in chrome and then nothing to my website related is shown (even 24 hours after the latest deployment). So, can I fix this? Or is this a general, permanent problem with subdomains in SEO?

This is my _docuument.js for now:
       
```
        <meta name='author' content='einKnuffy' />
        <meta name='publisher' content='einKnuffy' />
        <meta name='copyright' content='einKnuffy' />
        <meta
          name='description'
          content='DESCRIPTION...'
        />
        <meta
          name='keywords'
          content=SOME KEYWORDS....'
        />
        <meta name='page-topic' content='TOPIC....' />
        <meta name='page-type' content='FAQ' />
        <meta name='audience' content='Alle' />
        <meta httpEquiv='content-language' content='en' />
        <meta name='robots' content='index, follow' />
```

I would appreciate any help! Thanks, love it **<3**

**Comment by amyegan** [comment, marked as answer]
Hi @einKnuffy! The short answer is "SEO is tricky", but there are some things you can do to boost how well your site appears in search. 
- Go through the [Next.js SEO lessons](https://nextjs.org/learn/seo/introduction-to-seo) (seems like you already did this one!)
- Submit your sitemap on [Google Search Console](https://search.google.com/search-console/about) and [Bing Webmaster Tools](https://www.bing.com/webmasters/about) which should also get it picked up by DuckDuckGo and Yahoo. You should submit your website to [Baidu](http://zhanzhang.baidu.com/linksubmit/url) and [Yandex](http://webmaster.yandex.com/addurl.xml) if you're hoping for Chinese and Russian visitors.
- Check [Core Web Vitals](https://vercel.com/blog/core-web-vitals) and improve where necessary

Another thing to know is that **Preview** deployments are _not_ indexed by search engines because of an added `X-Robots-Tag: noindex` header. https://vercel.com/guides/are-vercel-preview-deployment-indexed-by-search-engines

**Reply by kxzo** [reply]
Thanks, I'll have a look and give you feedback whether it helped or not! 👍

**Reply by kxzo** [reply]
So according to Lighthouse in Chrome dev tools, my SEO is like 94 which isn't too bad. So. it should somehow be shown in the search engine but it isn't. So, I still have the problem, although I used Bing Webmaster Tools, checked Core Web Vitals, got through the SEO lessons and submited my sitemap to Bing Webmaster Tools.

**Reply by amyegan** [reply]
I really only know the basics, and it looks like you have that covered on your site. Submitting a site does not mean it will be indexed immediately. It can take a few days so it's probably too soon to expect any change yet.

If your keywords are very popular, then other sites may be showing up higher than yours for a variety of reasons. [Google's SEO Starter Guide](https://developers.google.com/search/docs/beginner/seo-starter-guide) goes over some tips for getting a site ranked higher.

If you're interested in learning more about SEO and boosting your site's search rankings, I would recommend finding a community targeted at that topic where you will find people with specialized expertise to help you.

**Reply by hsm207** [reply]
@amyegan i have the same question but my site is a docusaurus site. The [Next.js SEO lessons](https://nextjs.org/learn/seo/introduction-to-seo) isn't actually specific to Next.js sites only, is it?

Also, just wanted to confirm with you that Vercel isn't doing anything to prevent sites hosted under the `.vercel.app.` domain from being indexed by search engines, right?

**Reply by amyegan** [reply]
@hsm207 The Next.js SEO lessons have good general information, but the code examples in the lessons related specifically to Next. For more general information, I would recommend starting with [Google's SEO Starter Guide](https://developers.google.com/search/docs/fundamentals/seo-starter-guide), and then looking through the rest of the info found on that documentation site for more detailed strategies.

Vercel does not block production deployments on `*.vercel.app` domains from being indexed. But preview deployments are blocked from being indexed by search engines with the `x-robots-tag: noindex` header by default. This is to ensure your SEO ranking is not negatively impacted by duplicate content. You can [modify the header](https://vercel.com/guides/are-vercel-preview-deployment-indexed-by-search-engines#x-robots-tag-header) if you do want a preview deployment to be indexable
